### PR TITLE
[CELEBORN-1240][FOLLOWUP] Introduce Husky Configuration to Celeborn Web

### PR DESCRIPTION
**Pull request by tiny-dust**
### What changes were proposed in this pull request?

Removes the pre-commit of husky.

### Why are the changes needed?

Husky pre-commit is not necessary for git commit. There is contribution guide and CI check for the frontend code style.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Local test.

### CODE REVIEW

File: web/.husky/pre-commit [hunk 1]
```diff
@@ -17,6 +17,11 @@
 
 #!/usr/bin/env sh
 
+# When Web UI does not install dependencies, no git commit check is performed.
```
Comment (SteNicholas) [inline web/.husky/pre-commit:20]:
```suggestion
# Git commit does check when web module does not install dependencies.
```

**Review by labbomb — APPROVED** [review]
LGTM.

File: web/.husky/pre-commit [hunk 2]
```diff
@@ -17,6 +17,11 @@
 
 #!/usr/bin/env sh
 
+# Git commit does check when web module does not install dependencies.
```
Comment (SteNicholas) [inline web/.husky/pre-commit:20]: Could we remove this precommit check? IMO, the CI would check the frontend code style and the contribution guide introduces the way to reformat frontend code.

Comment (tiny-dust) [reply web/.husky/pre-commit]: ok.

**Review by SteNicholas — APPROVED** [review]

**Comment by SteNicholas** [comment]
LGTM. Thanks. Merged into main(v0.5.0).

**Pull request closed without merging**